access/slack/config.go

```diff
@@ -22,14 +22,21 @@ type SlackConfig struct {
 }
 
 const exampleConfig = `# example slack plugin configuration TOML file
```

Comment:
This is the exact string that's used to generate the example configuration file when running: `teleport-slack configure > teleport-slack.toml` or `teleport-slack configure > /path/to/teleport-slack.toml`. Tested this. 

Decided to include the commands and previous configuration.